Author: Sergei Golovan <sgolovan@debian.org>
Description: A quick hack which removes rpath from crypto and ssl applications binaries to
 satisfy Debian policy.
Forwarded: no
Last-modified:Sat, 27 Feb 2010 09:34:43 +0300

--- erlang-16.b-dfsg.orig/erts/configure.in
+++ erlang-16.b-dfsg/erts/configure.in
@@ -4301,9 +4301,9 @@
 SSL_CC_RUNTIME_LIBRARY_PATH=
 SSL_LD_RUNTIME_LIBRARY_PATH=
 SSL_DED_LD_RUNTIME_LIBRARY_PATH=
-cc_rflg="$CFLAG_RUNTIME_LIBRARY_PATH"
-ld_rflg="$LDFLAG_RUNTIME_LIBRARY_PATH"
-ded_ld_rflg="$DED_LD_FLAG_RUNTIME_LIBRARY_PATH"
+cc_rflg=
+ld_rflg=
+ded_ld_rflg=
 
 if test "$SSL_APP" != "" && test "$SSL_DYNAMIC_ONLY" = "yes" && \
    { test "$cc_rflg" != "" || test "$ld_rflg" != "" || test "$ded_ld_rflg" != ""; } ; then
